kolibri/deployment/default/urls.py

```diff
@@ -26,4 +26,6 @@
     url(r'^admin/', include(admin.site.urls)),
     url(r'^$', include('kolibri.core.urls')),
     url(r'', include('kolibri.content.urls')),
+    url(r'^api/', include('kolibri.auth.urls')),
```

Comment:
Another thing we need to decide: how do we structure our API endpoint URLs?

Could be `/api/<resourcename>` or `/api/<appname>/<resourcename>/`, for example.

The content API (CC: @66eli77) is mounted directly on `/`, but should probably at least also be mounted under `/api/`, but first the tests would need to be refactored to use `reverse`, because right now they're using hardcoded hand-constructed URLs.